streamlit_app.py

```diff
@@ -204,6 +205,7 @@ def get_page_content(
         streamlit_cloud_url: str,
         pypi_name: str,
         twitter_username: str,
+        support_page_username : str;
```

Comment:
Not specific to this PR, but it looks like a lot of these should be Optional

Reply:
Agreed! 👍🏻
Let me know what all other badges I can try adding here!  😊

Reply:
```suggestion
        support_page_username : str,
```

This prevents the app from running! Otherwise LGTM